packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/footer/disableAdd.spec.tsx

```diff
@@ -44,37 +44,38 @@ describe(disableAdd, () => {
       )
     ).toBeTruthy();
   });
-  it('should return true if KPI is selected and has a assets already added', () => {
+
+  it('should return false if status/kpi is selected and has NO assets already added', () => {
```

Comment:
This test name says status and kpi, but only tests status. Could you test both?

Reply:
I had for both originally, the test file was getting tooo big , and the logic is exactly the same for both, so decided to remove it.. let me add to the same test case